Koha/Plugin/EDS/bootstrap/modules/eds-detail.tt

```diff
@@ -1287,7 +1287,7 @@
 [% END %]
 [% IF ( OpacStarRatings != 'disable' ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.rating.js"></script>[% END %]

```

Comment:
Both on my test environment and ebsco-gss.net the jquery.rating.js and jquery.rating.css seems to fail.

Request URL:
http://koha-2311.ebsco-gss.net/opac-tmpl/bootstrap/lib/jquery/plugins/jquery.rating.js

When looking for OpacStarRatings, I only came up with usage such as this:

```
    [% IF ( Koha.Preference('OpacStarRatings') == 'all' ) %]
        [% Asset.js("lib/jquery/plugins/jquery.barrating.min.js") | $raw %]
        [% Asset.js("js/ratings.js") | $raw %]
    [% END %]
```
Looking at relevant directory https://github.com/Koha-Community/Koha/tree/23.11.x/koha-tmpl/opac-tmpl/lib/jquery/plugins also doesn't show that jquery.rating.js - something that while not touched in this PR, we probably should investigate

Reply:
Thank you for pointing this out, we are adding this to work planning for next PR.